certified-connectors/Leadinfo/apiDefinition.swagger.json

```diff
@@ -0,0 +1,293 @@
+{
+  "swagger": "2.0",
+  "info": {
```

Comment:
@GerbenZ01 please add the "contact" section within the "info" section. Please follow our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json).